### PR TITLE
Pr 3406

### PR DESCRIPTION
**Pull request by lissyx**

### CODE REVIEW

File: taskcluster/examples-mic_vad_streaming-py39.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  template_file: examples-base.tyml
+  docker_image: "python:3.9"
+  dependencies:
+    - "linux-amd64-cpu-opt"
+  system_setup:
+    >
+      apt-get -qq -y install portaudio19-dev pulseaudio
+  args:
+    tests_cmdline: "${system.homedir.linux}/DeepSpeech/examples/mic_vad_streaming/test.sh 3.9.0:"
+  metadata:
```
Comment (lissyx) [inline taskcluster/examples-mic_vad_streaming-py39.yml:11]: @dag7dev My bad, you need to add `workerType: "${docker.dsTests}"` here

Comment (lissyx) [reply taskcluster/examples-mic_vad_streaming-py39.yml:11]: @dag7dev You will also have to do that on `examples-mic_vad_streaming-py38.yml`

Comment (dag7dev) [reply taskcluster/examples-mic_vad_streaming-py39.yml:11]: Oh our friend at docker!
Do I need to do it for 3.8 file too?

Comment (lissyx) [reply taskcluster/examples-mic_vad_streaming-py39.yml:12]: Yes please. I've pushed the fix in https://github.com/mozilla/DeepSpeech/pull/3408/commits/5c0734e0c7d74d702d80410b99ae5e1ce25851f3 to be able to run the PR

**Review by dag7dev — APPROVED** [review]

**Pull request merged**